recipes/ifaa/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ifaa" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ifaa-{{ version }}.tar.gz
+  sha256: fc3f148a91e8cb03d8690fd78c8f70d27ff20633e1db3a13eec9bf8c6aead166
+
+build:
+  number: 0
+  skip: True  # [not unix]
```

Comment:
This seems like it could be noarch. If the only reason it is not is a missing dependency, the missing dependency can be ignored and it should still be noarch.